tools/cxbe/Main.cpp

```diff
@@ -54,6 +62,40 @@ int main(int argc, char *argv[])
         szXbeTitle[40] = '\0';
     }
 
+    // interpret title id
+    if(szXbeTitleID[0])
+    {
+        bool hex = true;
+        for(int i = 0; szXbeTitleID[i]; ++i)
+        {
+            if(szXbeTitleID[i] == '-')
+            {
+                hex = false;
+                break;
+            }
+        }
+        if(hex)
+        {
+            sscanf(szXbeTitleID, "%x", &dwTitleId);
+        }
+        else
+        {
+            char titlechar[2];
+            unsigned titleno;
```

Comment:
Using a not explicitly unsigned char can lead to incorrect results when shifting, better use the types defined in `Cxbx.h` or `stdint.h` for these variables.

Reply:
Will adding `unsigned` do?
I prefer to use stdlib types with an stdlib function